cookbook/offscreen-clipboard-write/offscreen.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
+<input id="text" type="text">
```

Comment:
A `textarea` is always used for this task because it preserves linebreaks inside the text.

Reply:
Good call, thanks!

```suggestion
<textarea id="text" type="text"></textarea>
```